sources/packages/api/src-sql/Institution/add-col-institution-type.sql

```diff
@@ -0,0 +1,13 @@
+-- Add column institution_type_id as we will be storing the institution type of each enstitution created/modified
```

Comment:
enstitution change to Institution

Reply:
Thanks for noticing @guru-aot . Typo has been fixed.